src/storage/exec/UpdateNode.h

```diff
@@ -765,6 +751,14 @@ class UpdateEdgeNode : public UpdateNode<cpp2::EdgeKey> {
         }
         // step 3, insert new edge data
         batchHolder->put(std::move(key_), std::move(nVal));
+
+        // extra phase: if there are some extra requirement.
+        for (auto& [k, v] : edgeContext_->kvAppend) {
```

Comment:
Make sure `kvAppend` and `kvErased` doen't have overlaps here. Because `doublePrime` in `ChainUpdateEdgeProcessorLocal` could be add to both of them here.

Reply:
either?

Reply:
Basically, you won't either write `doublePrime` or remove `doublePrime`, but if the batch contains both `write` and `remove`, how can we make sure which happens first?